pros/conductor/conductor.py

```diff
@@ -17,11 +18,11 @@
 
 MAINLINE_NAME = 'pros-mainline'
 MAINLINE_URL = 'https://purduesigbots.github.io/pros-mainline/pros-mainline.json'
-BETA_NAME = 'kernel-beta-mainline'
-BETA_URL = 'https://raw.githubusercontent.com/purduesigbots/pros-mainline/master/beta/kernel-beta-mainline.json'
+PROS4_NAME = 'kernel-4-mainline'
```

Comment:
Lets use a different naming scheme. This represents a depo and as of right now we call that depo the beta depo. I dont think `PROS4` would be the best way to reference the depo in code. I'm thinking about naming the current depo to something like `early access` so that references would look like `early_access_templates` or `depo[EARLY_ACCESS]`. Seems a lot more professional and will avoid future confusion.